--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -1333,6 +1333,10 @@ struct net_device {
 						 * avoid dirtying this cache line.
 						 */
 
+        struct net_device       *master; /* Pointer to master device of a group,
+                                          * which this device is member of.
+                                          */
+
 	/* Interface address info used in eth_type_trans() */
 	unsigned char		*dev_addr;	/* hw address, (before bcast
 						   because most packets are
